### DISCUSSION TITLE
Profile 页面 author_title 怎么换行？

### DISCUSSION THREAD

**Discussion by GanZhiXiong** [ppoffice/hexo-theme-icarus#922] [Q&A]
尝试了 https://github.com/ppoffice/hexo-theme-icarus/issues/37 提供的方法，还是不行。

**Comment by GanZhiXiong** [comment]
我知道可以直接在 `profile.jsx` 中修改，但是这样就失去了配置的意义了。

**Comment by ppoffice** [comment]
1. ```yaml
   author_title: |
       line 1
       line 2
    ```
2. 另外需要给相应的HTML标签加上`white-space:pre-wrap;`

**Reply by GanZhiXiong** [reply]
> 1. ```yaml
>    author_title: |
>        line 1
>        line 2
>    ```
> 2. 另外需要给相应的HTML标签加上`white-space:pre-wrap;`

这样写就可以实现了。👍👍👍

**Comment by leimao** [comment]
@ppoffice 这个2具体怎么做能说的详细一点吗？非常感谢。

**Comment by leimao** [comment]
这么做应该就可以了，虽然我并不是很懂HTML
```
{authorTitle ? <p style="white-space: pre" class="is-size-6 is-block">{authorTitle}</p> : null}
```

**Comment by leimao** [comment]
更确切的说应该是
```
{authorTitle ? <p style="white-space: pre-line" class="is-size-6 is-block">{authorTitle}</p> : null}
```

**Reply by taneemishere** [reply]
can you please tell me where should we write this statement...??

**Reply by frid** [reply]
@taneemishere try this

1. in _config.icarus.yml  
```       
author_title: |     
    line 1
    line 2
```
2.  edit profile.jsx under hexo-theme-icarus\layout\widget
replace
`{authorTitle ? <p class="is-size-6 is-block">{authorTitle}</p> : null}`
to
`{authorTitle ? <p style="white-space: pre-line" class="is-size-6 is-block">{authorTitle}</p> : null}`
3. clean and regenerate
